src/Entity/CropType.php

```diff
@@ -61,10 +66,48 @@ class CropType extends ConfigEntityBundleBase implements CropTypeInterface {
   public $description;
 
   /**
+   * The ratio of the image of this crop type.
+   *
+   * @var string
+   */
+  public $aspect_ratio;
+
+  /**
    * {@inheritdoc}
    */
   public function id() {
     return $this->id;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getAspectRatio() {
+    return $this->aspect_ratio;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validate() {
+    $violations = $this->getTypedData()->validate();
+    return new ConstraintViolationList(iterator_to_array($violations));
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getIterator() {
+    return new \ArrayIterator();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getCropTypeNames() {
+    return array_map(function ($bundle_info) {
```

Comment:
I am not sure if there is any coding standard related to this, but I prefer something like:

```
return array_map(
  function ($bundle_info) { return $bundle_info['label'];},
  \Drupal::entityManager()->getBundleInfo('crop')
);
```

It seems more readable to me.